src/link.js

```diff
@@ -0,0 +1,25 @@
+export function baseLink(element, parser, options) {
+  const {Link} = parser.minim.elements;
+  options = options || {}
+
+  const link = new Link();
+  link.relation = options.relation || 'origin';
+  link.href = `http://docs.apiary.io/validations/swagger#${(options.fragment || '')}`;
```

Comment:
We don't need to point them to anything, these are just links. That is all.

Also moving to help.apiary.io means @Baggz need to edit the editor too. So, I have left if for some later date.